collage/mainwindow.py

```diff
@@ -121,7 +129,14 @@ def open_text_window(self):
         return window.get_return()
 
     def add_photo(self, where):
-        pass
+        filename = filedialog.askopenfilename(
+            title="Select file",
+            filetypes=(
+                ("jpeg files", "*.jpg"),
+                ("png files", "*.png"),
```

Comment:
add another image formats

Reply:
Also, an oridnary user doesn't know about all these image extensions, he just wants to add some image. But now he must iterate through all the defined extensions unless he finds the desired one. So, it would be better to define filetypes as `('image files', ('.png', '.jpg'))` to make his life easier.

Reply:
These extensions are enough: `gif|jpg|jpeg|tiff|png|bmp`.